README.md

```diff
@@ -104,6 +122,10 @@ daily, rather than what is seen on the _omim.org/entry/MIM#_ pages. Note that th
 always in sync, and that one or the other may be slightly more up-to or out-of date for a period of time.
 
 ### `review.tsv`
+Currently, all of these review cases are part of the "D2G" (Disease-Gene) pipeline, and reflect cases where we have 
```

Comment:
Why are you adding another name, e.g. "D2G" (Disease-Gene) pipeline, when this is the OMIM Ingest Pipeline overall? I've been trying to use "ingest" to refer to the steps to prepare the external source for use in downstream pipelines, e.g. OMIM gene pipeline, Synonym Sync, etc.